structopt-derive/src/macros.rs

```diff
@@ -0,0 +1,3 @@
+macro_rules! my_quote {
+    ($($t:tt)*) => (quote_spanned!(::proc_macro2::Span::call_site() => $($t)*))
+}
```

Comment:
If you had a file, you should add the header. But maybe for that, just defining it in lib.rs is OK.